examples/View.elm

```diff
@@ -1,79 +1,165 @@
 module View exposing (view)
 
 import Time exposing (Time)
-import Html exposing (..)
-import Html.Events exposing (onClick)
-import Html.Attributes exposing (id, for, attribute, class, value)
-import Utils exposing (timeAgo)
-import Model exposing (Model, Record, Records, Msg(..))
-import Form
-import Dict
+import Html
+import Html.Events
+import Html.Attributes
+import Utils
+import Model exposing (Model, Record, Msg(..), Sort(..))
 
 
 formatLastModified : Int -> Time -> String
 formatLastModified timestamp currentTime =
-    timeAgo (toFloat timestamp) currentTime
+    Utils.timeAgo (toFloat timestamp) currentTime
 
 
-iconBtn : String -> Msg -> Html Msg
+iconBtn : String -> Msg -> Html.Html Msg
 iconBtn icon action =
-    button [ class "btn btn-xs btn-default", onClick action ]
-        [ i [ class ("glyphicon glyphicon-" ++ icon) ] [] ]
+    Html.button
+        [ Html.Attributes.class "btn btn-xs btn-default"
+        , Html.Events.onClick action
+        ]
+        [ Html.i [ Html.Attributes.class ("glyphicon glyphicon-" ++ icon) ]
+            []
+        ]
 
 
-recordRow : Time -> Record -> Html Msg
+recordRow : Time -> Record -> Html.Html Msg
 recordRow currentTime { id, title, description, last_modified } =
-    tr []
-        [ td [] [ text id ]
-        , td [] [ text (Maybe.withDefault "[empty]" title) ]
-        , td [] [ text (Maybe.withDefault "[empty]" description) ]
-        , td [] [ text (formatLastModified last_modified currentTime) ]
-        , td []
+    Html.tr []
+        [ Html.td [] [ Html.text id ]
+        , Html.td [] [ Html.text (Maybe.withDefault "[empty]" title) ]
+        , Html.td [] [ Html.text (Maybe.withDefault "[empty]" description) ]
+        , Html.td [] [ Html.text (formatLastModified last_modified currentTime) ]
+        , Html.td []
```

Comment:
I actually likes to have them prefixed so that we know it is part of the Html lib.
Also exposing the Html component directly will use a lot of the variable names.
However I know that in react it is exposed directly. A matter of taste maybe?